frontend/src/container/TriggeredAlerts/index.tsx

```diff
@@ -29,6 +34,18 @@ function TriggeredAlerts(): JSX.Element {
 		},
 	);
 
+	useEffect(() => {
+		if (!isEqual(responseRef.current, alertsResponse)) {
+			if (!alertsResponse.isLoading) {
```

Comment:
why such a handling ? if we want to log the event on the success of the response we can use `onSuccess` callback in the useQuery hook ?

Reply:
here the alertResponse API is happening multiple times on the page like polling, which makes the page refresh and attaching event based on onSuccess callback will behave as if user have again visited the page on every api refresh. hence attach it to only when there is an actual difference in response